Bottom: 8cf1fe7e564f969740ade3952b079bcb521d9c2d
Top:    d94f1e9e819297d97bd979b9c336e3199db760b3
Author: Ian Kent <ikent@redhat.com>
Date:   2013-09-23 14:44:44 +0800

Refresh of debug.patch

---

diff --git a/lib/rpc_subs.c b/lib/rpc_subs.c
index 57a136b..d1f8b82 100644
--- a/lib/rpc_subs.c
+++ b/lib/rpc_subs.c
@@ -636,7 +636,7 @@ int rpc_portmap_getport(struct conn_info *info,
 		pmap_info.port = rpc_getrpcbport(info->proto);
 		pmap_info.program = rpc_getrpcbyname(rpcb_prog);
 		pmap_info.version = rpcb_version;
-		error(LOGOPT_ANY, "program %d port %d version %d", pmap_info->program, pmap_info->port, pmap_info->version);
+		error(LOGOPT_ANY, "program %d port %d version %d", pmap_info.program, pmap_info.port, pmap_info.version);
 		pmap_info.proto = info->proto;
 		pmap_info.send_sz = RPCSMALLMSGSIZE;
 		pmap_info.recv_sz = RPCSMALLMSGSIZE;
